docker/7.4/.bashrc

```diff
@@ -86,4 +86,5 @@ alias cpl='zf && zg && cd ext/ && ./install && ..'
 alias codecept='php -d extension=ext/modules/phalcon.so ./vendor/bin/codecept '
 alias phpcs='php -d extension=ext/modules/phalcon.so ./vendor/bin/phpcs '
 alias phpcbf='php -d extension=ext/modules/phalcon.so ./vendor/bin/phpcbf '
+alias psalm='php -d extension=ext/modules/phalcon.so ./vendor/bin/psalm '
```

Comment:
```suggestion
alias psalm='php ./vendor/bin/psalm '
```

Psalm don't need `phalcon.so`, as it runs against `ide/` folder, where only PHP files.